FireboltNETSDK/Client/FireboltParameter.cs

```diff
@@ -56,7 +56,8 @@ public override DbType DbType
         {
             get
             {
-                var chType = FireboltDbType;
+                // not sure if this is right
+                var chType = FireboltDbType ?? global::FireboltDbType.String;
```

Comment:
We have the same behaviour in Python SDK. This shouldn't happen anyways, should it?

Reply:
You're right. What was worrying me here is that this class is used in parametrised queries, but there are checks for the correct type. Also, this represents db type for a given value, if value is not recognised it makes sense for it to default to string.